docs/modules/ROOT/pages/references/functional-requirements/billing.adoc

```diff
@@ -0,0 +1,165 @@
+= Billing and Reporting
+
+NOTE: This page is currently very VSHN specific and targets the https://products.docs.vshn.ch/products/appuio/cloud/pricing.html[APPUiO Cloud price model].
+
+Each organization gets an invoice, containing all items of the organization in a summarized format.
+Detailed information can be looked up in the {portal}.
+
+== Features
+
+A list of features the system should have:
+
+* Versioned price lists, including a default price list
+* Custom price plan per organization
+* Discounting
+* Pre-Payment / Credit Management / Credit Vouchers
+* Invoice payment by credit-card (Automated / Manually)
+* Access to current running costs (Dashboard)
+* Access to invoices (PDF)
+
+== Example Invoice
+
+.Example invoice for APPUiO Cloud
+====
+_Invoice period: 1.1. - 31.1. (44'640 minutes, timezone UTC)._
+
+[cols="4,1,1,1,1,1", options="header", stripes="hover"]
+|===
+|Description
+|Quantity
+|Unit of Measure
+|Unit Price
+|Discount
+|Total (Rounded)
+
+6+|*Zone: cloudscale.ch LPG 2 - Namespace: my-app-prod*
+
+a|Compute
+
+Min: 128 MiB +
+Max: 1034 MiB +
+Avg: 585.66 MiB
+
+|7028
+|MiB
+|CHF 0.0000012432
+|0 %
+>|CHF 0.01
+
+6+>|*Subtotal*: CHF 0.01
+
+6+|*Zone: Exoscale GVA 1 - Namespace: my-app-test*
+
+a|Compute
+
+Min: 302 MiB +
+Max: 3990 MiB +
+Avg: 1689.2 MiB
+
+|16892
+|MiB
+|CHF 0.0000009043
+|0 %
+>|CHF 0.02
+
+a|Persistent Storage (PVC) - RWO
+
+Avg usage: 10 GiB
+|100
+|GiB
+|CHF 0.000081
+|0 %
+>|CHF 0.01
+
+6+>|*Subtotal*: CHF 0.03
+
+6+>e|*Total*: *CHF 0.04*
+
+|===
+====
+
+See <<Data for Example Invoice>> below for where these numbers are coming from.
+
+== Data for ERP
+
+TIP: This is targeting Odoo.
+
+To be able to create an invoice in Odoo, a bunch of data needs to be prepared and the be sent to Odoo.
+
+=== Per Invoice
+
+Partner ID:: Corresponding to the Organization.
+Account ID:: Hardcoded to 49.
+Currency ID:: Hardcoded to 6 (CHF).
+Invoice date:: Date of the invoice.
+Journal ID:: Hardcoded to 1.
+Name:: Name of the invoice.
+Payment term:: Hardcoded to 3.
+State:: Draft.
+User ID:: The ID of the user who created the invoice.
+
+=== Per Line Item
```

Comment:
I remember we talked about not to use the Odoo internal price table but use our own. For this I am missing a field to enter the chared amount.

Reply:
Yes, that was missing. Added.